src/main/java/kernitus/plugin/OldCombatMechanics/OCMMain.java

```diff
@@ -3,6 +3,7 @@
 import kernitus.plugin.OldCombatMechanics.hooks.PlaceholderAPIHook;
 import kernitus.plugin.OldCombatMechanics.hooks.api.Hook;
 import kernitus.plugin.OldCombatMechanics.module.*;
+import kernitus.plugin.OldCombatMechanics.module.Module;
```

Comment:
Not necessary, make sure you organize imports as well before pushing code. You have a .* import on the line above so this import is doing nothing.